package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@evolv/asset-manager",
-  "version": "1.30.0",
+  "version": "1.30.1",
```

Comment:
I don't believe it will hurt anything, but you don't need bump this version. We have automation that increments the version when the repo gets deployed. (For this repo, the authoritative version is stored as a commit tag rather than in _package.json_.)

Reply:
Thanks for letting me know, I'll update the procedure doc to reflect that